crates/topos/tests/config.rs

```diff
@@ -24,6 +25,39 @@ async fn polygon_edge_path(path: &str) -> String {
     installation_path.to_str().unwrap().to_string()
 }
 
+async fn generate_polygon_edge_genesis_file(
```

Comment:
Not sure if we should just generate the `genesis.json` once and then copy it to the `temp_dir`?

Reply:
I like every test to be independent. They are executed in random order.